json/flagd-definitions.yaml

```diff
@@ -1,68 +1,85 @@
+$id: "https://flagd.dev/schema/flagd-definitions.json"
```

Comment:
is this still valid?

Reply:
Yes, that is the ID of the schema - it's basically a name. It's ideal if it's also the location of the schema, and the extension doesn't really matter.